src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.test.tsx

```diff
@@ -19,13 +20,12 @@ describe('<ProposalDataListItemStructure/> component', () => {
 
         const baseProps: Omit<IProposalDataListItemStructureProps, 'result'> = {
             date: new Date().toISOString(),
-            protocolUpdate: false,
-            publisher: { address: '0x123' },
+            tag: 'OSx Update',
+            publisher: { address: '0x123', profileLink: '#' },
```

Comment:
We can remove the `date` property as optional, right? Same for the `tag` and `profileLink` properties, let's only define the required properties here